src/hivpy/sexual_behaviour.py

```diff
@@ -277,3 +284,99 @@ def update_rred_balance(self, population):
         else:
             population.loc[men, col.RRED_BALANCE] = 1/rred_balance
             population.loc[women, col.RRED_BALANCE] = rred_balance
+
+    def update_ltp_rate_change(self, date):
+        if date1995 < date < date2000:
+            dt = diff_years(date1995, date)
+            self.ltp_rate_change = self.annual_ltp_rate_change**(dt)
+        elif date >= date2000:
+            self.ltp_rate_change = self.annual_ltp_rate_change**5
+            # don't really need to keep doing this update after 2000
```

Comment:
There is something similar in the HIV_status code, and there will probably be more. One idea (for the future!) could be to track the date only in the Population, and have modules "subscribe" to date changes, so they can update their internal parameters when needed (and ignore any date change after that).